tests/unit/service/test_resourcedriver.py

```diff
@@ -454,7 +454,7 @@ def test_get_lifecycle_execution_adopt_suspended(self):
         self.assertIsInstance(execution, LifecycleExecution)
         self.assertEqual(execution.request_id, 'Adopt::1::request123')
         self.assertEqual(execution.status, 'FAILED')        
-        self.assertEqual(str(execution.failure_details), 'failure_code: INFRASTRUCTURE_ERROR description: SUSPEND_COMPLETE')
+        self.assertEqual(str(execution.failure_details), 'failure_code: INFRASTRUCTURE_ERROR description: Adopt Failed. Stack Status: SUSPEND_COMPLETE Reason: SUSPEND_COMPLETE')
```

Comment:
This error message is still not very clear to me. Perhaps something like `Adopt failed: cannot adopt stack with status SUSPEND_COMPLETE` would read better?

Reply:
OK so not including the `stack_status_reason`  (which I agree can look a bit cumbersome)
and instead it is:

`Adopt failed: cannot adopt stack with status + <STACK_STATUS>`

yeah I agree with this - I did think the reason was a bit too wordy

Reply:
Done